### DISCUSSION TITLE
Make "Add single comment" the default option - not "Start a review" (in code review page)

### DISCUSSION THREAD

**Discussion by weitzhandler** [community/community#61067] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

Hi,

I've been facing this issue numerous times and always get annoyed by it.

When responding to PR review comments and want to reply with a comment, the reply button in green is "Start a review", meaning that when I click <kbd>Ctrl</kbd>+<kbd>Enter</kbd> by habit, it starts a review, so I then have to delete the comment and post it again as a comment.

![image](https://github.com/community/community/assets/2716316/23952bbf-a384-4616-91c6-97011273f32a)

Please swap the buttons and make the default button "Add single comment".  

If you still have a strong opinion about keep "Start a review" as default, "Add single comment" should at least be the default button when it doesn't make sense for it to be there, for example if I am the one who opened this PR, or I'm replying to existing comments of a previous already existing preview, I've just committed code and more instances.

**Comment by agribov** [comment]
+1 - Replying to comments is a common action that's done over and over, while starting a review is a conscious effort, that needs to just be started once, deliberately. Would make more sense for "comment" to be the default, or let us set this in settings.

**Comment by MatheusRich** [comment]
Related: Hide the "Start a review" button when commenting on your own PRs, since it's impossible to review your own PR. Let "Add single comment" be the primary action.

**Comment by Fruglemonkey** [comment]
100 times this - it's the git equivalent of stubbing my toe when doing a PR and makes me want to punch my face every time

**Comment by gitressa** [comment]
This could take a long time to get fixed ... In the meantime, at the very least, Github could give a hint when hovering over each button, describing what they do. Something as simple as adding a title field ... 

| <span class="Button-label" title="If your own PR, and commenting on a review, probably use this option.">Add single comment</span> | <span class="start-review-label" title="If not your own PR, and no comments yet exist, probably use this option.">Start a review</span> |
| --- | --- |

**Comment by djrodgerspryor** [comment]
I'd also be open to a per-user setting for this. I never use reviews (I want my comments to show up as I write them, not be hidden, since I'm often in a live convo with the author), but some people love them.

**Comment by thebravoman** [comment]
Could we please have this. Some configuration that we could change. I've forgotten to press "Finish review" of 100% of the PRs I've reviewed in the last year. At this point and just not reviewing them in github, but using slack, because there is always some pr that has not be "fisnished" 

There was also not a single time when I needed to add a text to the "Finish review"

**Comment by ensonic** [comment]
Please. There are so many PR reviews I have pending comments for. If you want to keep the "Start Review" button please remind me to finish the review, anything else hardly makes sense to me.
A stop-gap measure would be a setting where I can make swap the default review action.

**Comment by kkorotkov** [comment]
@weitzhandler, I think the title should be changed because it sounds misleading and says the opposite of what the request actually means (which I fully support).

**Reply by weitzhandler** [reply]
Changed title

**Comment by calebmeyer** [comment]
If you're willing to use a userscript (with tampermonkey, greasemonkey or any of the other addons) or a custom stylesheet (with stylish or any of the other addons), here's one that worked for me:
```js
// ==UserScript==
// @name         Disable Start a Review button
// @namespace    http://tampermonkey.net/
// @version      2025-02-28
// @description  Hides the start a review button so I don't accidentally click it
// @author       Caleb Meyer
// @match        https://github.*/*/pull/*/files
// @icon         none
// @grant        none
// ==/UserScript==

(function() {
    'use strict';

    let css = '#files .Button--primary:has(.start-review-label) { display: none; }',
    head = document.head || document.getElementsByTagName('head')[0],
    style = document.createElement('style');

    head.appendChild(style);

    style.type = 'text/css';
    style.appendChild(document.createTextNode(css));
})();
```

**Comment by evan-hughes** [comment]
It's really easy to forgot to finish a review and navigate away from a PR. Could there be a setting that would make "Add Single Comment" action the default?

**Comment by jpbochi** [comment]
I've been using my own userscript for this: https://github.com/jpbochi/user-scripts/blob/master/only-single-comments.js

It changes the default button, including styles and the behavior of `cmd+enter`. It observes page changes for new forms, so no action is needed to change the default at any point.

Unfortunately, GitHub is introducing a completely refactored PR review UI, and my userscript doesn't work on that. See https://github.com/orgs/community/discussions/163932 I sent some feedback about it: https://github.com/orgs/community/discussions/163932#discussioncomment-14002776

I tried to update my script, but their new code uses fancy JS code rather than standard web elements like `<form>`, `<input>`, `<button>`. Tampering with that one is way more complicated.

**Comment by bdaz** [comment]
Just another +1 for this. I do this daily and it's really irritating. 

The conservative scope for this change is to make "Comment" (not "Start a Review") the default on my own PRs (but leave things as-is on other PRs). You can't review your own PRs so the default behavior is purposelessly annoying.

**Reply by weitzhandler** [reply]
I'd want Comment to be the default in all PRs.

**Comment by npcastro** [comment]
+1 for this. The current behaviour is really frustrating.

**Comment by alighulamulhassan** [comment]
It took me a couple days realising that my team wasn't able to see my responses to their comments on my PR, since all 7 of my comments were Pending since I unbeknownst to me started a review - would appreciate the ability to set the default as comment only

**Comment by jbell-sr** [comment]
+1 to this. Replying to comments should not default start a review, at a minimum.

**Comment by Mats-SX** [comment]
+1 please make this possible

**Comment by simonhellweger** [comment]
+1

**Comment by jordan-klaviyo** [comment]
+1

**Comment by BrandonNoad** [comment]
+1

**Comment by kennyknechtorsi** [comment]
+1

**Comment by iamjwc** [comment]
+1
